### PR TITLE
fix(replay): fix rendering doctype + svg

### PR DESCRIPTION
**Pull request by blakebyrnes**
This PR fixes Replay so it can display namespaced attributes and elements (ie, svg).

It also moves testing to github actions and resolves a few issues on Ubuntu and Windows.

### CODE REVIEW

File: core/index.ts [hunk 1]
```diff
@@ -249,39 +253,55 @@ export default class Core implements ICore {
...
+      }
+      const window = session.window;
+      this.byWindowId[window.id] = new Core(session);
+      return {
+        sessionId: session.id,
+        sessionsDataLocation: session.baseDir,
+        windowId: window.id,
+        replayApiServer: this.replayServer?.url,
+      };
+    });
```
Comment (calebjclark) [inline core/index.ts:270]: Why are we putting this in a Queue?

Comment (blakebyrnes) [reply core/index.ts:270]: There are several async actions in each of the shutdown, startup, createsession, etc functions in here. We now additionally have a delayed "shutdown" when you close a session and you didn't manually start. When you shutdown and start frequently, it's created too many problems with a shutdown happening "mid-session creation" and other similar problems

Comment (blakebyrnes) [reply core/index.ts:270]: A queue is a strange name for this, but just checking a variable at the beginning of each function isn't working. Any ideas for a better name of the queue or a different pattern?

Comment (calebjclark) [reply core/index.ts:270]: Yeah, the naming isn’t very clear what’s its doing. I don’t have a better idea right now.

**Pull request merged**